manifests/profile/dashboards.pp

```diff
@@ -71,6 +73,7 @@
   String $telegraf_token_name = $puppet_operational_dashboards::telegraf_token_name,
   String $influxdb_token_file = $puppet_operational_dashboards::influxdb_token_file,
   Boolean $include_pe_metrics = $puppet_operational_dashboards::include_pe_metrics,
+  Boolean $manage_system_board = $puppet_opertational_dashboards::manage_system_board,
```

Comment:
Typo on this line causing catalog failues.

Reply:
Thanks for the review. Fixed.